src/nvim/testdir/test52.in

```diff
@@ -61,5 +60,5 @@ STARTTEST
 ENDTEST
 
 1 utf-8 text: Для Vim version 6.2.  Последнее изменение: 1970 Jan 01
-2 cp1251 text: ��� Vim version 6.2.  ��������� ���������: 1970 Jan 01
-3 cp866 text: ��� Vim version 6.2.  ��᫥���� ���������: 1970 Jan 01
+2 cp1251 text: ��� Vim version 6.2.  ��������� ���������: 1970 Jan 01
+3 cp866 text: ��� Vim version 6.2.  ��᫥���� ���������: 1970 Jan 01
```

Comment:
Fixed in https://github.com/neovim/neovim/pull/4808. Will merge after build passes.